acceptance-tests/apps/sqsapp/internal/app/redrive.go

```diff
@@ -0,0 +1,52 @@
+package app
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+
+	"sqsapp/internal/credentials"
+
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+)
+
+func handleRedrive(creds credentials.Credentials) func(r *http.Request) (int, string) {
+	return func(r *http.Request) (int, string) {
+		binding := r.PathValue("binding_name")
+		log.Printf("Handling redrive on binding %q\n", binding)
+
+		cred, ok := creds[binding]
+		if !ok {
+			return http.StatusBadRequest, fmt.Sprintf("no creds found for binding: %q", binding)
+		}
+		cfg, err := cred.Config()
+		if err != nil {
+			return http.StatusInternalServerError, fmt.Sprintf("could not read AWS config: %q", err)
+		}
+
+		dlqBinding := r.URL.Query().Get("dlq_binding")
```

Comment:
There's something a bit messy about taking one binding name as a URL path and another as a URL parameter, but I can't think of a better way, and this is just a test app, not a production-grade REST API...

Reply:
Definitely agree its ugly API design. Thinking out it we just need the ARN of the DLQ as a param which we actually have in the test. I will make a change to pass the DLQ ARN.